packages/mobile/src/send/SendAmount.tsx

```diff
@@ -206,6 +215,21 @@ export class SendAmount extends React.Component<Props, State> {
     }
   }
 
+  isPaymentLimitReached = (now: number) => {
```

Comment:
Can we add a unit test for this? It may be easier to do if you pull this method out of the component and move it to somewhere in `/utils`

Reply:
Moved to utils. Working on unit tests next.

Reply:
I also passed the recent payments as a parameter to make unit testing nicer.